persistence-kafka/src/main/scala/com/evolutiongaming/kafka/flow/kafkapersistence/KafkaPersistence.scala

```diff
@@ -0,0 +1,59 @@
+package com.evolutiongaming.kafka.flow.kafkapersistence
```

Comment:
How about `com.evolutiongaming.kafka.flow.persistence`, i.e. without `kafka`?

Reply:
The thing is I have a package object which defines an extension constructor `PartitionFlowOf.eagerRecoveryKafkaPersistence`, so the client will of using this extension is to be expressed by the import `com.evolutiongaming.kafka.flow.kafkapersistence._`. With this regard I'd like to keep the package name distinct.